.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -8,13 +8,29 @@ const isProd = JSON.parse(
 );
 const token = core.getInput('GITHUB_TOKEN', {required: true});
 const date = new Date();
-const message = `🚀 Deployed 🚀 to ${
-    isProd ? 'production' : 'staging'
-} on ${date.toDateString()} at ${date.toTimeString()}`;
 
 const octokit = github.getOctokit(token);
 const githubUtils = new GithubUtils(octokit);
 
+function getDeployTableMessage(platformResult) {
```

Comment:
NAB could do this instead:

```javascript
function getDeployTableMessage(platformResult) {
    switch (platformResult) {
        case 'success':
            return `${platformResult} ✅`;
        case 'cancelled':
            return `${platformResult} 🔪`;
        case 'skipped':
            return `${platformResult} 🚫`;
        case 'failure':
        default:
            return `${platformResult} ❌`;
    }
}
```

Also, you're missing a method doc comment.

Reply:
Yeah we could, I was thinking anything but a success is a ✅ - But more emojis the better, so let's do it